config/routes.rb

```diff
@@ -1,7 +1,21 @@
 Rails.application.routes.draw do
+
+  namespace :api do
+  namespace :v1 do
+    get 'users/create'
```

Comment:
Use 2 (not 0) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.